tests/modularity.py

```diff
@@ -4,10 +4,20 @@
 import subprocess
 import sys
 import time
+import difflib
 
 # Specify the directory for the binaries
 DIR_EXAMPLES = "build/examples"
 
+def print_string_diff(str_a, str_b):
+    for i,s in enumerate(difflib.ndiff(str_a, str_b)):
+        if s[0]==' ': continue
```

Comment:
Nit: please use some auto-formatting tool for the python, next time if this file will saved by me or others with auto-format configured, there will be unnecessary diff